src/App/App.tsx

```diff
@@ -19,6 +21,7 @@ const Donate = lazy(() => import("pages/Donate"));
 const DonateFiatThanks = lazy(() => import("pages/DonateFiatThanks"));
 const Gift = lazy(() => import("pages/Gift"));
 const DonateWidget = lazy(() => import("pages/DonateWidget"));
+const Login = lazy(() => import("pages/Login"));
```

Comment:
Let's also keep the internal naming and URL slug the same too, using "Signin".
```suggestion
const Signin = lazy(() => import("pages/Signin"));
```